.circleci/config.yml

```diff
@@ -15,9 +15,10 @@ test-install-dependencies: &test-install-dependencies
     sudo apt-get install -y mongodb-org-shell google-chrome-stable
 
 test-run: &test-run
-  name: Run Tests
-  command: |
-    for i in $(seq 1 5); do mongo rocketchat --eval 'db.dropDatabase()' && npm test && s=0 && break || s=$? && sleep 1; done; (exit $s)
+          name: Run Tests
+          command: |
+            if [[ $DISABLE_SMARTI ]]; then rm -rf ./tests/end-to-end/ui_smarti; fi;
```

Comment:
Smarti tests can currently not run on our CI due to memory limits. => These tests can be remove based on the ENV